src/core/or/circuitpadding.c

```diff
@@ -380,17 +380,17 @@ circpad_choose_state_length(circpad_machine_state_t *mi)
  */
 static circpad_delay_t
 circpad_distribution_sample_iat_delay(const circpad_state_t *state,
-                                      circpad_delay_t min_delay)
+                                      circpad_delay_t delay_shift)
```

Comment:
Maybe we should document the exact behavior of this argument in the doxygen comment

Reply:
Done in 32c821c47b.